ReactAndroid/src/main/java/com/facebook/react/ReactAndroidHWInputDeviceHelper.java

```diff
@@ -25,20 +25,19 @@
    * Contains a mapping between handled KeyEvents and the corresponding navigation event
    * that should be fired when the KeyEvent is received.
    */
-  private static final Map<Integer, String> KEY_EVENTS_ACTIONS = MapBuilder.of(
-    KeyEvent.KEYCODE_DPAD_CENTER,
-    "select",
-    KeyEvent.KEYCODE_ENTER,
-    "select",
-    KeyEvent.KEYCODE_SPACE,
-    "select",
-    KeyEvent.KEYCODE_MEDIA_PLAY_PAUSE,
-    "playPause",
-    KeyEvent.KEYCODE_MEDIA_REWIND,
-    "rewind",
-    KeyEvent.KEYCODE_MEDIA_FAST_FORWARD,
-    "fastForward"
-  );
+  private static final Map<Integer, String> KEY_EVENTS_ACTIONS = new HashMap<>();
```

Comment:
You can keep using MapBuilder here:

private static final Map<Integer, String> KEY_EVENTS_ACTIONS = MapBuilder.of()
.put(KeyEvent.KEYCODE_DPAD_CENTER, "select")
....
.build();